### PR TITLE
Feat/slow stop

### PR DESCRIPTION
**Pull request by tkhmy**
## Description(required)
- add slow stop view

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: src/signage/src/signage/route_handler.py [hunk 1]
```diff
@@ -72,10 +73,12 @@ def emergency_checker_callback(self):
         if self._parameter.ignore_emergency:
             in_emergency = False
         else:
-            in_emergency = (
-                self._autoware.information.mrm_behavior == MrmState.EMERGENCY_STOP
-                or self._autoware.information.mrm_behavior == MrmState.COMFORTABLE_STOP
-            )
+            in_emergency = self._autoware.information.mrm_behavior == MrmState.EMERGENCY_STOP
```
Comment (yabuta) [inline src/signage/src/signage/route_handler.py:76]: すみません、意図を伝えきれてなかったですが減速中は「減速します」という表示はそのままにしてもらってもいいでしょうか？

「減速します」と表示する条件
```
self._autoware.information.mrm_behavior == MrmState.COMFORTABLE_STOP
and self._autoware.information.motion_state == MotionState.MOVING
```
「安全確認のため停車中です」と表示する条件
```
self._autoware.information.mrm_behavior == MrmState.COMFORTABLE_STOP
and self._autoware.information.motion_state == MotionState.STOPPED
```

**Review by yabuta — CHANGES_REQUESTED** [review]

File: src/signage/src/signage/route_handler.py (hunk 1 above)
Comment (tkhmy) [reply src/signage/src/signage/route_handler.py:76]: なるほど、２つのパターンになりますね、修正します。

Comment (tkhmy) [reply src/signage/src/signage/route_handler.py:77]: [2be2237](https://github.com/tier4/AutowareSignage/pull/77/commits/2be2237875cbaa637be810389701ae1c3fd5c1c8) で修正しました。

**Review by yabuta — APPROVED** [review]
LGTM

**Pull request merged**